client-app/app/styles/app.css

```diff
@@ -1,5 +1,5 @@
 body {
-  font-family: "Roboto", Arial, "Liberation Sans", "DejaVu Sans", sans-serif;
```

Comment:
hmmm I recall Jeff wanted Roboto, any reason you want it removed?

Reply:
I made this change for the new settings page which looked really weird with Roboto (it looks the same even with a really low font-weight)
![Screenshot_20190318-084113](https://user-images.githubusercontent.com/17474474/54509402-7b897100-495a-11e9-89fe-c39584ebcf9f.jpg)

Compare this with how Roboto looks on desktop:

![image](https://user-images.githubusercontent.com/17474474/54509492-c7d4b100-495a-11e9-81a0-94f3d15bc50d.png)


I didn't remove Roboto completely, the "homepage" still looks the same, I just moved Roboto from `body` to specific elements: https://github.com/discourse/logster/pull/82/files/a9aead48dc135cdee6b0000e8897bb8a3dc9ba09#diff-fa37db806cc56aa352948366c4a7a53cR37

Reply:
cool, makes sense.